ttnn/cpp/ttnn/operations/eltwise/binary/binary_pybind.hpp

```diff
@@ -105,6 +105,9 @@ void bind_binary_operation(py::module& module, const binary_operation_t& operati
 
         Supports broadcasting.
 
+        Note:
```

Comment:
what is this?

Reply:
This section will have information like supported types, layouts etc.. which will be passed as one of the parameter